modules/large-deployments/pages/hub-install.adoc

```diff
@@ -1,14 +1,27 @@
 [[lsd-hub-install]]
 = Hub Installation
 
-Before you begin, you need to install the XMLRPC API package, and configure the Hub Server to use the API.
+Before you begin, you need to install the ``hub-xmlrpc-api`` package, and configure the Hub Server to use the API.
 
 
 
-.Procedure: Installing and Configuring the XMLRPC API
+.Procedure: Installing and Configuring the Hub XMLRPC API
 . On the Hub Server, or on a host that has access to all peripheral servers' XMLRPC APIs, install the ``hub-xmlrpc-api`` package.
 The package is available in the {productname} {productnumber} repositories.
-. Verify if the configuration file available at  ``/etc/hub/hub-xmlrpc-api-config.json`` is pointing to the correct Hub instance and modify it as required.
-. OPTIONAL: In the configuration file, check the settings for the timeout parameters ``connect_timeout`` and ``read_write_timeout``.
-The values for these parameters are defined in seconds.
-. Restart services to pick up the configuration changes.
+. OPTIONAL: the Hub XMLRPC API service does not start on installation. You can set it up to start automatically with systemd, by executing the following commands
```

Comment:
```suggestion
. OPTIONAL: Set the Hub XMLRPC API service to start automatically on installation:
+
```

Reply:
Hmm, the intended message is "with the commands below the service will start on each boot (`enable`) and now (`start`)" - merely installing the rpm package gives you neither.

Would the following wording work?

```suggestion
. OPTIONAL: Set the Hub XMLRPC API service to start automatically at boot time, and start it immediately:
```

Reply:
sure :)